internal/pkg/cli/flag.go

```diff
@@ -278,10 +279,12 @@ To use it for an ECS service, specify --generate-cmd <cluster name>/<service nam
 Alternatively, if the service or job is created with Copilot, specify --generate-cmd <application>/<environment>/<service or job name>.
 Cannot be specified with any other flags.`
 
-	vpcIDFlagDescription           = "Optional. Use an existing VPC ID."
-	publicSubnetsFlagDescription   = "Optional. Use existing public subnet IDs."
-	privateSubnetsFlagDescription  = "Optional. Use existing private subnet IDs."
-	certsFlagDescription           = "Optional. Apply existing ACM certificates to the internet-facing load balancer."
+	vpcIDFlagDescription              = "Optional. Use an existing VPC ID."
+	publicSubnetsFlagDescription      = "Optional. Use existing public subnet IDs."
+	privateSubnetsFlagDescription     = "Optional. Use existing private subnet IDs."
+	certsFlagDescription              = "Optional. Apply existing ACM certificates to the internet-facing load balancer."
+	internalALBSubnetsFlagDescription = `Optional. Specify subnet placement for an internal load balancer.
+Cannot be specified with --default-config or any of the --override flags.`
```

Comment:
Should we say something about the default? and that the input is IDs
```suggestion
	internalALBSubnetsFlagDescription = `Optional. Specify subnet IDs for an internal load balancer.
By default, the load balancer will be placed in your private subnets.
Cannot be specified with --default-config or any of the --override flags.`
```